policy-controller/k8s/status/src/index.rs

```diff
@@ -78,7 +79,7 @@ pub struct Index {
 
     /// Maps HttpRoute ids to a list of their parent and backend refs,
```

Comment:
We'll want to update this comment to indicate that both HttpRoutes and GrpcRoutes are stored in this map.

Reply:
Updated the docstring as indicated